src/financial-reports/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: layouts/default.njk
```

Comment:
Not Blocking: If we use the `title:` front-matter value in the markdown files it will be added to the layout's `<title>` tag, as well as an `<h1>` in the pages header.